### PR TITLE
daveroga/rationalize configs

### PR DESCRIPTION
**Pull request by daveroga**
Fixes issue #355 .

The configuration for this application is becoming fragmented. It now exists in:

- Some constants.mjs files.
- Environment files
- config module files
- env-cmd module files

Probably it is sensible just to use one of these.

Steps to solve this:
- Used config module files for tests and removed constants.mjs.
- Removed env-cmd module files.
- Tried to use config module in cli but then found some problems with babel/webpack and the wallet. Some constants are cli specific and should be in the cli project.

### CODE REVIEW

**Comment by daveroga** [comment]
- Used config module files for tests and removed constants.mjs.
- Removed env-cmd module files.
- Tried to use config module in cli but then found some problems with babel/webpack and the wallet. Couldn't still solve it.

File: config/default.js [hunk 1]
```diff
@@ -126,4 +126,74 @@ module.exports = {
...
+      optimistWsUrl: '',
+      web3WsUrl: '',
+    },
+    ropsten: {
+      name: 'Ropsten',
+      chainId: 3,
+      clientApiUrl: 'https://client1.testnet.nightfall3.com',
+      optimistApiUrl: 'https://optimist1.testnet.nightfall3.com',
+      optimistWsUrl: 'wss://optimist1-ws.testnet.nightfall3.com',
+      web3WsUrl: 'wss://ropsten1-ws.testnet.nightfall3.com',
```
Comment (druiz0992) [inline config/default.js:144]: Can we remove web3WsUrl from code?

**Review by druiz0992 — CHANGES_REQUESTED** [review]
I would like to keep web3 node url out of repo code. PR #334 proposes a way to do it. I suggest once it is approved, just reuse it

File: config/default.js (hunk 1 above)
Comment (daveroga) [reply config/default.js]: It has been removed in last commit `fix: remove testnet Ropsten URL from config`

**Comment by daveroga** [comment]
PR already solved but e2e test failing due to issue #427  that should be solved first.

**Comment by daveroga** [comment]
ganache-test seems failing sometimes. Forced modify a file to test it again.

**Comment by ChaitanyaKonda** [comment]
This seems to add 263 commits in total. Most of which are commits that already exist in the master.
Merging this has the risk of breaking the code and adding back logic that we removed. 

Could you revert this to just the commits from this PR please?

**Comment by daveroga** [comment]
Wrong commits added after rebase solved.

**Review by signorecello — APPROVED** [review]
LGTM but just mind some tests are being removed by #490

**Pull request merged**